MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/WorkspacePresenter/QWorkspaceDockView.h

```diff
@@ -129,7 +124,7 @@ class EXPORT_OPT_MANTIDQT_MANTIDWIDGETS QWorkspaceDockView
   void showWorkspaceData() override;
   void saveToProgram() override;
   void showInstrumentView() override;
-  void plotSpectrum(bool showErrors) override;
+  void plotSpectrum(std::string type) override;
```

Comment:
Function is called like this ``  view->plotSpectrum("Simple");`` so can't use`` string&``.